api/client/cli.go

```diff
@@ -162,6 +164,13 @@ func NewDockerCli(in io.ReadCloser, out, err io.Writer, clientFlags *cli.ClientF
 		}
 		cli.configFile = configFile
 
+		// Set the CLI's API version. This is a power-user feature to
+		// allow the user to set it to anything they want, typically for
+		// debugging or testing purposes.
+		if ver := os.Getenv("DOCKER_API_VERSION"); ver != "" {
+			api.Version = version.Version(ver)
```

Comment:
Since this is typically a CLI thing I wouldn't want any additional output to be send to the screen, mucking with what people see to the screen.